hyperspy/misc/utils.py

```diff
@@ -793,6 +793,10 @@ def stack(signal_list, axis=None, new_axis_name='stack_element',
         If mmap_dir is not None, and stack and mmap are True, the memory
         mapped file will be created in the given directory,
         otherwise the default directory is used.
+    load_to_memory : bool
+        If True loads all data to memory.
+        If False only loads the data upon request.
+        If None loads to memory
```

Comment:
As far as I can tell, this keyword is very similar to `mmap`, but is only used for h5py datasets?